tests/integration/test_properties.py

```diff
@@ -215,15 +215,39 @@ def test_delete_property_by_id(client, auth_headers, test_database):
     assert response == 404
 
 
-def test_update_property_by_id(client, auth_headers, create_property, test_database):
+def test_update_property(client, empty_test_db, valid_header, create_property):
```

Comment:
These tests seem like they belong in the Unit tests for the schemas since they are testing the validation on name.

Reply:
oh it looks like there are unit tests too, so I don't think we need these integration tests.